backend/src/main/java/c/team/quiz/model/Answer.java

```diff
@@ -11,4 +11,8 @@
     private String text;
     @Parameter(required = true)
     private boolean correct;
+
+    public Answer(String text){
+        this.text = text;
+    }
```

Comment:
Czyli jednak frontend wysyła samą odpowiedź bez sprawdzania poprawności, tak?

Reply:
Tak, tak przyjąłem, bo poprawność definiuje się przy tworzeniu quizu, więc dla zamkniętych da się to znaleźć, a dla otwartych to nie ma sensu trochę :D